### DISCUSSION TITLE
Add function/method parameters in trace frames to Ignition/Flare

### DISCUSSION THREAD

**Discussion by rubenvanassche** [spatie/flareapp.io-roadmap#71] [Ideas]
See the blogpost on the Flare blog: https://flareapp.io/blog/57-sensitive-parameters-in-php-82

**Comment by lloricode** [comment]
+1

**Comment by AlexVanderbist** [comment]
https://flareapp.io/blog/58-php-stack-trace-arguments-have-landed-in-flare 🥳
